synapse/res/templates/sso_redirect_confirm.html

```diff
@@ -2,35 +2,50 @@
     <meta name="viewport" content="width=device-width, user-scalable=no">
+    <link rel="stylesheet" type="text/css" href="sso.css">
     <style type="text/css">
       {% include "sso.css" without context %}
+
+      .confirm-trust {
+        margin: 34px 0;
+        color: #8D99A5;
+      }
+      .confirm-trust strong {
+        color: #17191C;
+      }
+
+      .confirm-trust::before {
+        content: "";
+        background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgiIGhlaWdodD0iMTgiIHZpZXdCb3g9IjAgMCAxOCAxOCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0xNi41IDlDMTYuNSAxMy4xNDIxIDEzLjE0MjEgMTYuNSA5IDE2LjVDNC44NTc4NiAxNi41IDEuNSAxMy4xNDIxIDEuNSA5QzEuNSA0Ljg1Nzg2IDQuODU3ODYgMS41IDkgMS41QzEzLjE0MjEgMS41IDE2LjUgNC44NTc4NiAxNi41IDlaTTcuMjUgOUM3LjI1IDkuNDY1OTYgNy41Njg2OSA5Ljg1NzQ4IDggOS45Njg1VjEyLjM3NUM4IDEyLjkyNzMgOC40NDc3MiAxMy4zNzUgOSAxMy4zNzVIMTAuMTI1QzEwLjY3NzMgMTMuMzc1IDExLjEyNSAxMi45MjczIDExLjEyNSAxMi4zNzVDMTEuMTI1IDExLjgyMjcgMTAuNjc3MyAxMS4zNzUgMTAuMTI1IDExLjM3NUgxMFY5QzEwIDguOTY1NDggOS45OTgyNSA4LjkzMTM3IDkuOTk0ODQgOC44OTc3NkM5Ljk0MzYzIDguMzkzNSA5LjUxNzc3IDggOSA4SDguMjVDNy42OTc3MiA4IDcuMjUgOC40NDc3MiA3LjI1IDlaTTkgNy41QzkuNjIxMzIgNy41IDEwLjEyNSA2Ljk5NjMyIDEwLjEyNSA2LjM3NUMxMC4xMjUgNS43NTM2OCA5LjYyMTMyIDUuMjUgOSA1LjI1QzguMzc4NjggNS4yNSA3Ljg3NSA1Ljc1MzY4IDcuODc1IDYuMzc1QzcuODc1IDYuOTk2MzIgOC4zNzg2OCA3LjUgOSA3LjVaIiBmaWxsPSIjQzFDNkNEIi8+Cjwvc3ZnPgoK');
+        background-repeat: no-repeat;
+        width: 24px;
+        height: 24px;
+        display: block;
+        float: left;
+      }
     </style>
 </head>
     <body>
         <header>
-            {% if new_user %}
-            <h1>Your account is now ready</h1>
-            <p>You've made your account on {{ server_name }}.</p>
-            {% else %}
-            <h1>Log in</h1>
-            {% endif %}
-            <p>Continue to confirm you trust <strong>{{ display_url }}</strong>.</p>
+            <h1>Continue to your account</h1>
         </header>
         <main>
-            {% if user_profile.avatar_url %}
-            <div class="profile">
+            <div class="profile{% if user_profile.avatar_url %} with-avatar{% endif %}">
+                {% if user_profile.avatar_url %}
                 <img src="{{ user_profile.avatar_url | mxc_to_http(64, 64) }}" class="avatar" />
-                <div class="profile-details">
-                    {% if user_profile.display_name %}
+                {% endif %}
+                {% if user_profile.display_name %}
                     <div class="display-name">{{ user_profile.display_name }}</div>
-                    {% endif %}
-                    <div class="user-id">{{ user_id }}</div>
-                </div>
+                {% else %}
+                    <div class="display-name">{{ user_id.split(":")[1:] }}</div>
+                {% endif %}
+                <div class="user-id">{{ user_id }}</div>
             </div>
```

Comment:
Should we split this out to a profile template if we're using it in multiple places?

Reply:
I thought of it but wasn't sure how to pass the variables.

Reply:
done in 7db023ac9b